doc/source/whatsnew/v0.21.1.txt

```diff
@@ -22,8 +22,8 @@ Other Enhancements
 ^^^^^^^^^^^^^^^^^^
 
 - :meth:`Timestamp.timestamp` is now available in Python 2.7. (:issue:`17329`)
--
--
+- :attr:`Timestamp.month_name`, :attr:`DatetimeIndex.month_name`, and :attr:`Series.dt.month_name` are now available (:issue:`12805`)
+- ``month_name` and ``weekday_name`` attributes will now return locale aware values (:issue:`12806`)
```

Comment:
move the 2nd point to bug fix; only mention weekday_name and use attr (as u did for month_name)